src/test/scala/org/camunda/feel/impl/SuppressedFailuresTest.scala

```diff
@@ -0,0 +1,115 @@
+package org.camunda.feel.impl
+
+import org.camunda.feel.api._
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.matchers.{MatchResult, Matcher}
+
+class SuppressedFailuresTest extends AnyFlatSpec
```

Comment:
The `UNKNOWN` is the default case. I plan to replace the default cases with a concrete failure type. And finally, remove the `UNKNOWN` case.

So, I would not write any specific test for it. It would be removed eventually. :wink: